assets/bingo/lang/de_de.json

```diff
@@ -1,17 +1,286 @@
+	"bingo.lobby.settings.player.save.description": "Voreinstellung überschreiben:",
+	"bingo.lobby.settings.player.save.new": "Erzeuge neue Voreinstellung",
+	"bingo.lobby.settings.player.save.new.explanation": "Setze den Namen der neuen Voreinstellung, indem Du den Nametag, den Du gerade erhalten hast, im Amboss umbenennst.\nWenn Du fertig bist, lass den Nametag einfach fallen.\nWenn dein Inventar voll ist, kannst Du einfach kurzfristig ein Item vor den Amboss werfen. Du nimmst es wieder auf, wenn der Nametag im Amboss ist.",
+	"bingo.lobby.settings.player.save.new.success": "Neue Voreinstellung “%s” erfolgreich gespeichert",
+	"bingo.lobby.settings.player.save.overwrite.success": "Voreinstellung “%s” erfolgreich mit neuen Einstellungen überschrieben.",
+
+	"bingo.lobby.settings.player.load.sign.line1": "Spieler",
+	"bingo.lobby.settings.player.load.sign.line2": "",
+	"bingo.lobby.settings.player.load.sign.line3": "Auswahl / Ansicht",
+	"bingo.lobby.settings.player.load.sign.line4": "Voreinstellung",
+	"bingo.lobby.settings.player.load.title": "Lade Spieler Voreinstellung",
+	"bingo.lobby.settings.player.load.description": "Zu ladende Voreinstellung:",
+	"bingo.lobby.settings.player.load.new.success": "Voreinstellung “%s” erfolgreich geladen",
+
+	"bingo.lobby.settings.player.configuration.empty": "Leer",
+	"bingo.lobby.settings.player.configuration.night_vision": "Nachtsicht",
+	"bingo.lobby.settings.player.configuration.boats": "Nachtsicht + 'Boats Boats Boats'",
+	"bingo.lobby.settings.player.configuration.elytra": "Nachtsicht + Elytra",
+	"bingo.lobby.settings.player.configuration.frostwalker": "Nachtsicht + Eisläufer",
+	"bingo.lobby.settings.player.configuration.underwater": "Nachtsicht + Unterwasseratmung + Wasserläufer",
+
+	"bingo.lobby.settings.preferences.sign.line1": "",
+	"bingo.lobby.settings.preferences.sign.line2": "Einstellungen",
+	"bingo.lobby.settings.preferences.sign.line3": "ändern",
+	"bingo.lobby.settings.preferences.sign.line4": "",
+
+	"bingo.lobby.settings.automatic_pregen.sign.line1": "Landschaft",
+	"bingo.lobby.settings.automatic_pregen.sign.line2": "vorzeitig",
+	"bingo.lobby.settings.automatic_pregen.sign.line3": "erzeugen",
+	"bingo.lobby.settings.automatic_pregen.sign.on": "AN",
+	"bingo.lobby.settings.automatic_pregen.sign.off": "AUS",
+	"bingo.lobby.settings.automatic_pregen.info": "Standardmäßig wird die Landschaft rund um den Spawn erzeugt, sobald Du auf das \"Spiel starten\" Schild klickst. Wenn diese Einstellung auf \"AN\" steht, beginnt die Erzeugung 5 Sekunden nach der Generierung der Bingo-Karte. Falls eine neue Karte genriert wird, wird eine laufende Erzeugung unterbrochen.\nBeachte bitte, dass dies Lag in der Lobby verursachen kann, was in einigen Fällen zu Verzögerung in der Funktionalität führt.",
+	"bingo.lobby.settings.automatic_pregen.turn_on": "Einschalten",
+	"bingo.lobby.settings.automatic_pregen.turn_off": "Ausschalten",
```

Comment:
I have now done said change for English. These are the source strings now:

```json
"bingo.lobby.settings.pregen_behavior.sign.line1": "Terrain pre-",
"bingo.lobby.settings.pregen_behavior.sign.line2": "generation",
"bingo.lobby.settings.pregen_behavior.sign.line3": "",
"bingo.lobby.settings.pregen_behavior.sign.on": "On game start",
"bingo.lobby.settings.pregen_behavior.sign.off": "On card gen",
"bingo.lobby.settings.pregen_behavior.info.on_game_start.prefix": "On game start: ",
"bingo.lobby.settings.pregen_behavior.info.on_game_start": "Terrain is pre-generated around spawn as soon as you hit the “Start Game” sign. This results in an additional wait time after starting the game.",
"bingo.lobby.settings.pregen_behavior.info.on_card_generation.prefix": "On card generation: ",
"bingo.lobby.settings.pregen_behavior.info.on_card_generation": "Pre-generation will automatically start 5 seconds after generating a new card. This may cause lag in the lobby while it is happening, making some functionallity in the lobby slow to react and unresponsive in some cases.",
"bingo.lobby.settings.pregen_behavior.turn_on": "Switch to “On game start”",
"bingo.lobby.settings.pregen_behavior.turn_off": "Switch to “On card generation”"
```

I also already pulled the current master branch into this branch, so you have the up-to-date version.